lib/type_check/builtin/compound_fixed_map.ex

```diff
@@ -70,12 +70,22 @@ defmodule TypeCheck.Builtin.CompoundFixedMap do
       |> color(:map, opts)
     end
 
+    if function_exported?(Macro, :inspect_atom, 2) do
+      # Elixir 1.14+
+      defp inspect_as_key(key) do
+        Macro.inspect_atom(:key, key)
+      end
+    else
+      # Legacy Elixir
+      import Code.Identifier, only: [inspect_as_key: 1]
+    end
+
```

Comment:
This is a nice way to solve the problem! :green_heart:

Reply:
Actually it doesn't work... Looks like Elixir is checking the `import` in 1.14 even if it is not run. Reverted PR to use `defdelegate` which was my original version